pr-data.csv

```diff
@@ -1031,6 +1031,52 @@ https://github.com/amzn/amazon-instant-access-sdk-java,3aae2253569751eaa4c419300
 https://github.com/amzn/amazon-instant-access-sdk-java,3aae2253569751eaa4c419300cf9915f03a42bf1,.,com.amazon.dtasdk.v3.messages.InstantAccessRequestTest.testSerialize,ID,Opened,https://github.com/amzn/amazon-instant-access-sdk-java/pull/13,
 https://github.com/amzn/amazon-pay-api-sdk-java,4093fc0ddca8e410764919f587a2e984246d5dc1,.,com.amazon.pay.api.GenerateButtonSignatureTest.testButtonSignatureWithJSONObject,ID,Opened,https://github.com/amzn/amazon-pay-api-sdk-java/pull/17,
 https://github.com/amzn/amazon-pay-api-sdk-java,ff493ba434744f19c76ee6b71b8035460c173fce,.,com.amazon.pay.api.UtilTest.testEnhanceResponseWithShippingAddressList,ID,Opened,https://github.com/amzn/amazon-pay-api-sdk-java/pull/21,
+https://github.com/apache/activemq,dfd35d89f62558881bda1ed4cf7a165502c5c989,activemq-stomp,org.apache.activemq.transport.stomp.auto.StompAutoNioSslTest.testTransformationNotOverrideSubscription,ID,Accepted,https://github.com/apache/activemq/pull/1129,
```

Comment:
What is the squashed commit for https://github.com/apache/activemq/pull/1129? The PR itself was "Closed", but the developer mentioned manually squashing commits.